tmva/sofie/inc/TMVA/RModel.hxx

```diff
@@ -17,6 +17,9 @@ private:
    int fVerbose = 0;
    int fBatchSize = -1;
    long fReadPos = 0;  // reading file position
+   size_t fConstantTensorSize = 0;
+   size_t fWeightsTensorSize = 0;
+   size_t fOtherTensorSize = 0;
```

Comment:
maybe better to document what `Other tensor` mean here?

Reply:
Done, thanks for the suggestion